### PR TITLE
Send to epp server

### PR DESCRIPTION
**Pull request by scottaubrey**
relates to https://github.com/elifesciences/enhanced-preprints-issues/issues/393

### CODE REVIEW

**Review by E-Kemp — CHANGES_REQUESTED** [review]
Not sure if this is my error, docmap-ts doesn't seem to have an @types definition

The following error happened after a docker compose up
```
src/activities/parse-docmap.ts(1,10): error TS2305: Module '"@elifesciences/docmap-ts"' has no exported member 'parser'.
```

**Comment by nlisgo** [comment]
> My guess would be to clear you local docker cache in case you have a cached, older version of docmap-ts library that's not being updated? Happy to pair on it if you still have problems after that.

I encountered the same issue that @E-Kemp did but removing the corresponding image and the container that was based on it, cleared that issue for me.

File: src/activities/send-version-to-epp.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  published?: Date,
+};
+
+type EPPImportResponse = {
+  result: boolean,
+  message: string,
+};
+
+export const sendVersionToEpp = async (versionJSON: EnhancedArticle): Promise<boolean> => {
+  const versionImportUri = `${config.eppServerUri}/import-version`;
```
Comment (nlisgo) [inline src/activities/send-version-to-epp.ts:46]: This endpoint isn't working for me. I know it isn't in the master branch of server yet. I can't see a corresponding PR on server. We should aim to get this endpoint in server ahead of this.

**Review by nlisgo — DISMISSED** [review]
Let's focus on introducing the `import-version` endpoint in server ahead of this.

**Comment by scottaubrey** [comment]
@nlisgo said:
> Let's focus on introducing the `import-version` endpoint in server ahead of this.

The endpoint did exist already mid-January. @E-Kemp moved it in the reorg a bit back, but it's still there. This branch probably needs updating to POST to `/preprints`:

```
>  kubectl port-forward -n epp--prod deployment/epp-server 3000:3000
Forwarding from 127.0.0.1:3000 -> 3000
Forwarding from [::1]:3000 -> 3000

<another tab> 
> curl -X POST http://localhost:3000/preprints
{"result":false,"message":"body sent failed validation: (ValidationError): \"id\" is required. \"msid\" is required. \"doi\" is required. \"versionIdentifier\" is required. \"article\" is required. \"preprintDoi\" is required. \"preprintUrl\" is required. \"preprintPosted\" is required"}

**Comment by nlisgo** [comment]
I almost happy with this. Just going to do some digging around server to verify that we have all the data we need to accept the import once the validation violations are resolved.

**Review by E-Kemp — APPROVED** [review]

**Pull request merged**